### DISCUSSION TITLE
Change WDTHCG, WDTHTH value and the effect about model stability

### DISCUSSION THREAD

**Discussion by newl74** [NOAA-EMC/WW3#1056] [Q&A]
When I simulate the storm wave effect, I got GSE. For this problem, I used PRO3 option and changed WDTHCG and WDTHTH value to 8.0. I wish to know the model stability is decreased or not if WDTHCG and WDTHTH value is higher than default value(1.5).

**Comment by aliabdolali** [comment]
It shouldn't affect the stability of the model but it would damp the peaks if you increase GSE parameters substantially. This is a known issue with the GSE correction.

**Comment by newl74** [comment]
Thank you for your answer.
Have a nice day.

-----Original Message-----
